config.yml

```diff
@@ -32,6 +73,78 @@ domains:
       - name: Remove device
         description: Unregister a device
         query: mutation.removeDevice
+      - name: IPMI Command
+        description: Run an IPMI command against a device (deprecated)
+        query: mutation.ipmiCommand
+
+  - name: Federation
+    description: View, create and manage guest accounts and federation hosts
+    usecases:
+      - name: Federation Guests
+        description: List all Federation Guests within an org
+        query: query.federationGuests
+      - name: Federation Guest
+        description: Fetch a single Federation Guest
+        query: query.federationGuest
+      - name: Federation Hosts
+        description: List all Federation Hosts within an org
+        query: query.federationHosts
+      - name: Federation Host
+        description: Fetch a single Federation Host
+        query: query.federationHost
+      - name: Edit Federation
+        description: Edit a federation
+        query: mutation.editFederation
+      - name: Create Federation Guest
+        description: Create a Federation Guest User
+        query: mutation.createFederationGuest
+      - name: Edit Federation Guest
+        description: Edit an existing Federation Guest User
```

Comment:
I don't think the word "user" is right, as it implies a human. Maybe "account" instead?